.github/workflows/build-collector.yml

```diff
@@ -0,0 +1,27 @@
+name: "Manual Build (Collector)"
+
+on:
+  workflow_dispatch:
```

Comment:
Looks useful.  I assume this can be run on any arbitrary fork/branch.  I suggest updating the collector documentation to highlight this as a way to build a custom layer.  Even better if you can include documentation for how to manually upload the layer to an account.  These are optional suggestions though and can be done later if you prefer.

Reply:
@tylerbenson 

I am planning to enhance this workflow (for ex. specifying build tags so custom collector binary will be able to built, deploying as layer to the specified AWS account, etc ...). 

So I think, it is better to merge this as is into the main branch and then I can test in in my fork/branch for the further enhancements easier. And then, I will add the final documentation based on those.